--- a/configure.sh
+++ b/configure.sh
@@ -366,8 +366,6 @@
     return 1
 }
 
-check_libdvbcsa_all
-
 #       _            _                   _   _   _
 #   ___| | ___   ___| | __     __ _  ___| |_| |_(_)_ __ ___   ___
 #  / __| |/ _ \ / __| |/ /    / _` |/ _ \ __| __| | '_ ` _ \ / _ \
@@ -1031,7 +1031,7 @@
 
 \$(APP): $APP_OBJS \$(CORE_OBJS) \$(MODS_OBJS)
 	@echo "BUILD: \$@"
-	@\$(LD) \$^ -o \$@ \$(LDFLAGS)
+	@\$(CC) \$^ -o \$@ \$(LDFLAGS)
 	@\$(OBJCOPY) --only-keep-debug \$@ \$@.debug
 	@\$(STRIP) $APP_STRIP_ARGS \$@
 
