--- a/src/libntirpc/ntirpc/netconfig.h
+++ b/src/libntirpc/ntirpc/netconfig.h
@@ -8,7 +8,7 @@
 #if defined(_WIN32)
 #define NETCONFIG "c:\\etc\\netconfig"
 #else
-#define NETCONFIG "/etc/netconfig"
+#define NETCONFIG "/opt/etc/netconfig"
 #endif
 #define NETPATH	  "NETPATH"
 
--- a/src/MainNFSD/nfs_rpc_dispatcher_thread.c
+++ b/src/MainNFSD/nfs_rpc_dispatcher_thread.c
@@ -1171,13 +1171,13 @@ void nfs_Init_svc(void)
 	netconfig_udpv4 = (struct netconfig *)getnetconfigent("udp");
 	if (netconfig_udpv4 == NULL)
 		LogFatal(COMPONENT_DISPATCH,
-			 "Cannot get udp netconfig, cannot get an entry for udp in netconfig file. Check file /etc/netconfig...");
+			 "Cannot get udp netconfig, cannot get an entry for udp in netconfig file. Check file /opt/etc/netconfig...");
 
 	/* Get the netconfig entries from /etc/netconfig */
 	netconfig_tcpv4 = (struct netconfig *)getnetconfigent("tcp");
 	if (netconfig_tcpv4 == NULL)
 		LogFatal(COMPONENT_DISPATCH,
-			 "Cannot get tcp netconfig, cannot get an entry for tcp in netconfig file. Check file /etc/netconfig...");
+			 "Cannot get tcp netconfig, cannot get an entry for tcp in netconfig file. Check file /opt/etc/netconfig...");
 
 	/* A short message to show that /etc/netconfig parsing was a success */
 	LogFullDebug(COMPONENT_DISPATCH, "netconfig found for UDPv4 and TCPv4");
@@ -1188,13 +1188,13 @@ void nfs_Init_svc(void)
 	netconfig_udpv6 = (struct netconfig *)getnetconfigent("udp6");
 	if (netconfig_udpv6 == NULL)
 		LogInfo(COMPONENT_DISPATCH,
-			"Cannot get udp6 netconfig, cannot get an entry for udp6 in netconfig file. Check file /etc/netconfig...");
+			"Cannot get udp6 netconfig, cannot get an entry for udp6 in netconfig file. Check file /opt/etc/netconfig...");
 
 	/* Get the netconfig entries from /etc/netconfig */
 	netconfig_tcpv6 = (struct netconfig *)getnetconfigent("tcp6");
 	if (netconfig_tcpv6 == NULL)
 		LogInfo(COMPONENT_DISPATCH,
-			 "Cannot get tcp6 netconfig, cannot get an entry for tcp in netconfig file. Check file /etc/netconfig...");
+			 "Cannot get tcp6 netconfig, cannot get an entry for tcp in netconfig file. Check file /opt/etc/netconfig...");
 
 	/* A short message to show that /etc/netconfig parsing was a success
 	 * for ipv6
