--- ./yandexwebdav.py.orig	2014-02-16 19:26:46.454030577 +0400
+++ ./yandexwebdav.py	2014-02-16 19:26:51.822045412 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/opt/bin/python
 # coding=utf-8
 
 import os
--- ./ydw.py.orig	2014-02-16 19:27:57.638030092 +0400
+++ ./ydw.py	2014-02-16 19:28:03.366029174 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/bin/python
 # -*- coding: utf-8 -*-
 import sys
 
