--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = po src test scripts contrib/caldav contrib/vdir
+SUBDIRS = src test scripts contrib/caldav contrib/vdir
 
 if ENABLE_DOCS
 SUBDIRS += doc
--- a/configure.ac
+++ b/configure.ac
@@ -153,7 +153,7 @@
 #                                                               Create Makefiles
 #-------------------------------------------------------------------------------
 AC_OUTPUT(Makefile doc/Makefile src/Makefile test/Makefile \
-          scripts/Makefile po/Makefile.in po/Makefile \
+          scripts/Makefile \
           contrib/caldav/Makefile contrib/vdir/Makefile)
 #-------------------------------------------------------------------------------
 #                                                                        Summary
