--- ./configure.ac.orig	2016-03-30 12:52:19.000000000 +0300
+++ ./configure.ac	2016-03-30 14:36:09.317681994 +0300
@@ -119,7 +119,7 @@
 	else
 		want_perl=yes
 		perl_prefix_note=no
-		PERL_MM_PARAMS="INSTALLDIRS=perl LIB=$withval"
+		PERL_MM_PARAMS="INSTALLDIRS=perl INSTALLPRIVLIB=$withval INSTALLARCHLIB=$withval INSTALLSITELIB=$withval INSTALLSITEARCH="
 		perl_library_dir="PERL_USE_LIB"
 		perl_set_use_lib=yes
 	fi,
@@ -367,70 +367,6 @@
 if test "$want_perl" != "no"; then
 	AC_MSG_CHECKING(for working Perl support)
 
-	if test -z "$perlpath"; then
-		perl_check_error="perl binary not found"
-	else
-		PERL_CFLAGS=`$perlpath -MExtUtils::Embed -e ccopts 2>/dev/null`
-	fi
-
-	if test "x$ac_cv_prog_gcc" = "xyes" -a -z "`echo $host_os|grep 'bsd\|linux'`"; then
-		dnl * several systems have Perl compiled with native compiler
-		dnl * but irssi is being compiled with GCC. Here we try to
-		dnl * fix those command line options a bit so GCC won't
-		dnl * complain about them. Normally there's only few options
-		dnl * that we want to keep:
-		dnl * -Ddefine -Uundef -I/path -fopt -mopt
-		PERL_CFLAGS=`echo $PERL_CFLAGS | $perlpath -pe 's/^(.* )?-[^DUIfm][^ ]+/\1/g; s/^(.* )?\+[^ ]+/\1/g'`
-
-		PERL_EXTRA_OPTS="CCCDLFLAGS=\"-fPIC\""
-		AC_SUBST(PERL_EXTRA_OPTS)
-	fi
-
-	if test -z "$PERL_CFLAGS"; then
-		if test -n "$perl_check_error"; then
-			perl_check_error="Error getting perl CFLAGS"
-		fi
-		AC_MSG_RESULT([not found, building without Perl])
-		want_perl=no
-	else
-		PERL_LDFLAGS=`$perlpath -MExtUtils::Embed -e ldopts 2>/dev/null`
-
-		dnl * remove all database stuffs
-		dnl * nsl is already in ldflags
-		dnl * libc is of course linked without needing -lc
-		dnl * -rdynamic must not be in LIBADD line
-		for word in -ldb -ldbm -lndbm -lgdbm -lc -rdynamic; do
-		  PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e "s/$word //" -e "s/$word$//"`
-		done
-
-		case "$host_os" in
-		  linux*)
-		    PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e 's/-lposix //' -e 's/-lposix$//'`
-		    ;;
-		  hpux*)
-		    if test "x$ac_cv_prog_gcc" = "xyes"; then
-		      PERL_CFLAGS=`echo $PERL_CFLAGS | $sedpath -e 's/-Ae //' -e 's/-Ae$//'`
-		      PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e 's/-Ae //' -e 's/-Ae$//'`
-		    fi
-		    ;;
-		  *)
-		    ;;
-		esac
-
-		dnl * check that perl's ldflags actually work
-		echo "#include <EXTERN.h>" > conftest.c
-		echo "#include <perl.h>" >> conftest.c
-		echo "int main(){perl_alloc(); return 0;}" >> conftest.c
-		$CC $CFLAGS $PERL_CFLAGS conftest.c -o conftest $LDFLAGS $PERL_LDFLAGS 2> perl.error.tmp > /dev/null
-		if test ! -s conftest -a "x$ignore_perl_errors" = "x"; then
-			perl_check_error="Error linking with perl libraries: $PERL_LDFLAGS: `cat perl.error.tmp`"
-			AC_MSG_RESULT([error linking with perl libraries, building without Perl])
-			want_perl=no
-		fi
-
-		rm -f perl.error.tmp
-	fi
-
 	if test "x$want_perl" != "xno"; then
 		AC_MSG_RESULT(ok)
 
@@ -464,15 +400,6 @@
 			PERL_STATIC_LIBS=0
 		fi
 
-		# figure out the correct @INC path - we'll need to do this
-		# through MakeMaker since it's difficult to get it right
-		# otherwise.
-		if test "x$perl_set_use_lib" = "xyes"; then
-			perl -e 'use ExtUtils::MakeMaker; WriteMakefile("NAME" => "test", "MAKEFILE" => "Makefile.test");' $PERL_MM_PARAMS >/dev/null
-			PERL_USE_LIB=`perl -e 'open(F, "Makefile.test"); while (<F>) { chomp; if (/^(\w+) = (.*$)/) { $keys{$1} = $2; } }; $key = $keys{INSTALLARCHLIB}; while ($key =~ /\\$\((\w+)\)/) { $value = $keys{$1}; $key =~ s/\\$\($1\)/$value/; }; print $key;'`
-			rm -f Makefile.test
-		fi
-
 		AC_SUBST(perl_module_lib)
 		AC_SUBST(perl_static_lib)
 		AC_SUBST(perl_module_fe_lib)
