--- a/scripts/exim_install
+++ b/scripts/exim_install
@@ -217,9 +217,7 @@
   # The exim binary is handled specially
 
   if [ $name = exim${EXE} ]; then
-    exim="./exim -bV -C /dev/null"
-    version=exim-`$exim 2>/dev/null | \
-      awk '/Exim version/ { OFS=""; print $3,"-",substr($4,2,length($4)-1) }'`${EXE}
+    version=exim-4.89
 
     if [ "${version}" = "exim-${EXE}" ]; then
       echo $com ""
